person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(id, age, name = 'unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] the `id` is not going to be passed as an argument, it should be assigned by the program, a good way to solve this is assigning a random number to the instance variable `@id`.